src/lib/numbers.ts

```diff
@@ -118,6 +118,20 @@ export const formatAmountFree = (amount: BigNumberish, tokenDecimals: number, di
   return trimZeroDecimals(amountStr);
 };
 
+export function formatAmountWithDollarSign(
+  amount: BigNumberish | undefined,
+  tokenDecimals: number,
+  displayDecimals?: number,
+  useCommas?: boolean
+) {
+  const amountStr = formatAmount(amount, tokenDecimals, displayDecimals, useCommas);
+  if (amount && bigNumberify(amount)?.gt(0)) {
+    return "$" + amountStr;
+  } else {
+    return "-$" + amountStr.slice(1);
+  }
+}
```

Comment:
Oh, i've added some formstters in the synthetics branch, If you'd like,  you can review this and maybe copy it here to standartize?

https://github.com/gmx-io/gmx-interface/pull/397/files#diff-7fdf1a5171d1d1dbaacd892cb9454006354046c1f481ef39d068a04eed7afb25